odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/reactive/ReactiveSearchEntrypointRepositoryImpl.java

```diff
@@ -515,8 +515,8 @@ public Mono<Integer> updateDatasetFieldSearchVectors(final long datasetFieldId)
             .and(DATASET_VERSION.VERSION.eq(dsvMaxField))
             .join(DATASET_STRUCTURE).on(DATASET_STRUCTURE.DATASET_VERSION_ID.eq(DATASET_VERSION.ID))
             .join(DATASET_FIELD).on(DATASET_FIELD.ID.eq(DATASET_STRUCTURE.DATASET_FIELD_ID))
-            .leftJoin(LABEL_TO_DATASET_FIELD).on(LABEL_TO_DATASET_FIELD.DATASET_FIELD_ID.eq(DATASET_FIELD.ID))
-            .leftJoin(LABEL).on(LABEL.ID.eq(LABEL_TO_DATASET_FIELD.LABEL_ID)).and(LABEL.DELETED_AT.isNull());
+            .leftJoin(TAG_TO_DATASET_FIELD).on(TAG_TO_DATASET_FIELD.DATASET_FIELD_ID.eq(DATASET_FIELD.ID))
```

Comment:
You forgot to use tags above at 
`final Field<String> labelName = LABEL.NAME.as("label_name");`